src/components/help/index.js

```diff
@@ -12,37 +13,37 @@ import styles from './styles.module.scss';
 export const Help = () => {
   return (
     <section className={styles.cards_wrapper}>
-      <a href="/" className={styles.card_link}>
+      <Link to="/getting_started" className={styles.card_link}>
         <div className={styles.card_wrapper}>
           <FaClipboardList />
           <h2>Getting Started</h2>
```

Comment:
It's not a good case to use `<h2>` more than 2 times in app, pls rewrite it to `<h3>`
`<h1>` - 1 time
`<h2>` - 2 time
h3,h4,h5,h6 - it doesn't matter
If this is just a mock component, and you will rewrite this code later, ok.